src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx

```diff
@@ -1,12 +1,95 @@
 import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { useState } from 'react'
+
+import Autocomplete from '@mui/material/Autocomplete'
+import TextField from '@mui/material/TextField'
+import SubjectsStepImage from '~/assets/img/subjects-step/image.svg'
 
 import { styles } from '~/containers/tutor-home-page/subjects-step/SubjectsStep.styles'
+import {categoriesMock, languagesMock} from '~/containers/tutor-home-page/subjects-step/constants'
+import AppButton from '~/components/app-button/AppButton'
+import { useStepContext } from '~/context/step-context'
+import AppChipList from '~/components/app-chips-list/AppChipList'
 
 const SubjectsStep = ({ btnsBox }) => {
+  const [category, setCategory] = useState(null)
+  const [subject, setSubject] = useState(null)
+  const { stepData, handleStepData } = useStepContext()
+  const subjectLabel = 'subjects';
+  const selectedSubjects = stepData[subjectLabel]
+
+  const handleChangeCategory = (e, categoryValue) => {
+    setCategory(categoryValue)
+    setSubject(null)
+  }
+
+  const handleChangeSubject = (e, subjectValue) => {
+    setSubject(subjectValue)
+  }
+
+  const handleAddSubject = () => {
+    handleStepData(subjectLabel, [...selectedSubjects, subject])
+    setSubject(null)
+  }
+
+  const handleDeleteSubject = (subjectName) => {
+    handleStepData(
+      subjectLabel,
+      selectedSubjects.filter((subject) => subject.name !== subjectName)
+    )
+  }
+  
   return (
     <Box sx={styles.container}>
+      <Box
+        alt='Girl studying'
+        component='img'
+        src={SubjectsStepImage}
+        sx={styles.image}
+      />
       <Box sx={styles.rigthBox}>
-        Subjects step
+        <Box sx={styles.titleWithForm}>
+          <Typography>
+            Velit officia consequat duis enim velit mollit. Other categories you can add in your account settings later.
```

Comment:
Can you use localization instead of this? You can find the necessary file in `constants/translations/become-tutor.json`. To do this, you need to use the `useTranslation()` hook.